File: program/print-ascii-value-of-a-string/print_character_of_an_ascii_value.go
package main

import "fmt"

func main() {
  var asciiValue int
  fmt.Println("Enter ASCII value:")
  fmt.Scan(&asciiValue)
	character := rune(asciiValue)

	fmt.Printf("Character corresponding to ASCII Code %d = %c\n", asciiValue, character)
}

File: program/print-character-of-an-ascii-value/PrintCharacterOfAnAsciiValue.java
import java.util.Scanner;
public class AsciiValue {

	public static void main(String[] args)
	{

	Scanner sc = new Scanner(System.in);
	System.out.println("Enter the Character:");
  char ch = sc.next().charAt(0);
int asciiVal= ch;
		System.out.println("The ASCII value  is: " + asciiVal);
	}
}
